pallets/passkey/src/weights.rs

```diff
@@ -44,20 +44,20 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
 	fn validate() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `39`
+		//  Measured:  `52`
 		//  Estimated: `5078`
-		// Minimum execution time: 42_000_000 picoseconds.
-		Weight::from_parts(43_000_000, 5078)
+		// Minimum execution time: 988_000_000 picoseconds.
+		Weight::from_parts(1_002_000_000, 5078)
```

Comment:
This is a massive jump and worrying. We might be able to optimize it a bite but not a lot. In worst case we might need to move that p256 signature check into extrinsic itself